build.proj

```diff
@@ -199,6 +199,11 @@
     <RemoveDir Directories='$([System.IO.Directory]::GetDirectories(".",".nuget", SearchOption.AllDirectories))' />
   </Target>
 
+  <Target Name="CleanTests">
```

Comment:
I would clean up all the projects under the test folder from the build directory.